docker-compose.yml

```diff
@@ -1,13 +1,32 @@
 version: '3.8'
 
 services:
+
+  my-mongodb:
+    image: mongo:3.7
+    container_name: mongodb
+    ports:
+      - 27017:27017
+    environment:
+      # MONGO_INITDB_DATABASE: test
+      MONGO_INITDB_ROOT_USERNAME: sdx
+      MONGO_INITDB_ROOT_PASSWORD: passw
+      MONGODB_DATA_DIR: /data/db
+    volumes:
+      # seeding scripts
+      # - ./mongo-entrypoint:/docker-entrypoint-initdb.d
+      # named volumes
+      - /home/cwang/mongodb:/data/db
```

Comment:
We should use something more portable than `/home/cwang/mongodb` here.

Reply:
Yeah, changed to ~/mongodb.

Reply:
This should be fine, it should changed by whomever is deploying the code. And also these will later go into the manifest, which will not be checked into git.